Purchases/Public/RCPurchases.m

```diff
@@ -737,7 +739,8 @@ - (void)handleRestoreReceiptPostWithInfo:(RCPurchaserInfo *)info
 - (void)checkTrialOrIntroductoryPriceEligibility:(NSArray<NSString *> *)productIdentifiers
                                  completionBlock:(RCReceiveIntroEligibilityBlock)receiveEligibility
 {
-    [self receiptData:^(NSData *data) {
+    [self.receiptFetcher receiptDataWithRefreshPolicy:RCReceiptRefreshPolicyOnlyIfEmpty
+                                           completion:^(NSData * _Nullable data) {
```

Comment:
slightly more verbose, but I actually prefer it this way - I've had multiple instances where I see `receiptData` and wonder how it handles refreshing, I think being more explicit here is helpful.